docs/docs/standard-lib/object.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Object
+nav_order: 17
+parent: Standard Library
+---
+
+# Object
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Object
+
+To make use of the Object module an import is required.
+
+```cs
+import Object;
+```
+
+### Object.getClassRef(string)
+
+This method will attempt to get a class reference from the class name provided as a string.
+
+Returns a Result and unwraps to an Object upon success.
+
+**NOTE:** This currently only works if the class is defined in the global scope
+
+```cs
+class Test {}
+
+Object.getClassRef("Test").unwrap(); // <Cls Test>
```

Comment:
Is there an easy way to get the name of the class from an object? I know we have a getAttributes method but I don't believe it includes the name of the class. That might be a nice addition so this can be even more dynamic.

Reply:
When you say class name, do you mean an actual reference to the class or do you mean the string representation of the class name?

Reply:
The string representation of the class.

Reply:
Ah there's currently no way to turn a class reference into a string'd name. We could add a name `_name` attribute similar to `_class` to classes

Reply:
That would be cool.

Reply:
Should be added as part of #543 

Now it would be:
```
class Test {}

print(Test._name);
print(Test()._class._name);
```

Reply:
Very cool!